src/cpp/rtps/messages/CDRMessage.cpp

```diff
@@ -19,7 +19,7 @@
  *
  */
 
-#include "CDRMessage.hpp"
+#include <rtps/messages/CDRMessage.hpp>
```

Comment:
I'd revert this change. Pre-processor is faster the other way, and they are side by side